src/native/util/os_darwin.cpp

```diff
@@ -37,12 +39,20 @@ get_current_uid()
 
 const uid_t ThreadScope::orig_uid = getuid();
 const gid_t ThreadScope::orig_gid = getgid();
+const std::vector<gid_t> ThreadScope::orig_groups = get_process_groups();
+
 
 void
 ThreadScope::change_user()
```

Comment:
Why only in Linux we have a function description? (on the same function `change_user`).

Reply:
I can add mac description as well